views/login.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>MozIoT Gateway Login</title>
+    <link rel="manifest" href="../app.webmanifest">
+    <link rel="icon" href="../images/icon.png" type="image/png" />
+    <link rel="stylesheet" type="text/css" href="/css/app.css" />
+    <link rel="stylesheet" type="text/css" href="/css/login.css" />
+  </head>
+  <body>
+    <img id="wordmark" src="../images/moziot-wordmark.png" />
+    <form id="login-form" action="/login" method="post">
+      <input type="email" name="email" placeholder="Email"></input>
+      <input type="password" name="password" placeholder="Password"></input>
+      <button id="login-button" type="submit">Log in</button>
+    </form>
+  </body>
+</html>
```

Comment:
It would be nice if this had a "Remember Me" checkbox, so that I don't have to re-enter the username and password everytime (probably best to add this as a separate PR)